.github/workflows/ci.yml

```
@@ -0,0 +1,186 @@
+        with:
+          toolchain: "1.98.0"
+          components: rustfmt, clippy
+
+      - uses: Swatinem/rust-cache@v2
+
+      - uses: extractions/setup-just@v4
+
+      - run: just versions
+
+      - name: Confirm the pinned toolchain is the one that's running
+        run: rustc --version | grep -qF ' 1.98.0 ' || { echo "expected rustc 1.98.0, got: $(rustc --version)" >&2; exit 1; }
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)